wagtail/admin/views/account.py

```diff
@@ -138,10 +139,17 @@ def render(self):
 
 class NameEmailSettingsPanel(BaseSettingsPanel):
     name = "name_email"
-    title = gettext_lazy("Name and Email")
     order = 100
     form_class = NameEmailForm
 
+    @cached_property
+    def title(self):
+        from wagtail.admin.views.account import email_management_enabled
```

Comment:
This is a function in the same file so I don't think this import is needed

Reply:
I'll do a PR to adjust this and merge.

Reply:
https://github.com/wagtail/wagtail/pull/10986